var/spack/repos/builtin/packages/h5z-zfp/package.py

```diff
@@ -66,3 +70,8 @@ def build_targets(self):
     def install_targets(self):
         make_args = ["install"]
         return make_args + self.make_defs
+
+    @run_after("build")
```

Comment:
I'm guessing this should be after("install") - will defer it to experts @adamjstewart to confirm..

Reply:
`make check` usually comes after `make` and before `make install` so this looks right